assets/components/forms/standardFields/input.scss

```diff
@@ -0,0 +1,10 @@
+@import './formFields';
+
+.component-input:not([type="radio"]) {
+  @include form-field;
+
+  height: 38px;
+  padding-left: 12px;
+  padding-right: 12px;
+  width: 314px;
```

Comment:
Could this be max-width instead? Just to allow it some flexibility in smaller cases